dev-system/support/build-for-container.sh

```diff
@@ -5,46 +5,55 @@
 #  - together: build and run dev-system in a container
 #  - default: pass no parameters, and the script will build and run dev-system
 
+build-base () {
+  cd ../common-app-lib
+  docker build -t cmr-base .
+  cd ../dev-system
+}
+
 build-and-run-container () {
   lein uberjar
   docker build -t $1 .
-  docker run -d $1 -p $2:$2
+  docker run -d -p $2:$2 $1
 }
 
+build-base
+
 if [[ $1 == "separate" ]]; then
-    cd ../../cubby-app
-    build-and-run-container "cubby" 3007
+    cd ../cubby-app
+    build-and-run-container "cmr-cubby" 3007
 
     cd ../metadata-db-app
-    build-and-run-container "metadata-db" 3001
+    build-and-run-container "cmr-metadata-db" 3001
 
     cd ../access-control-app
-    build-and-run-container "access-control" 3011
+    build-and-run-container "cmr-access-control" 3011
 
     cd ../search-app
-    build-and-run-container "search" 3003
+    build-and-run-container "cmr-search" 3003
 
     cd ../bootstrap-app
-    build-and-run-container "bootstrap" 3006
+    build-and-run-container "cmr-bootstrap" 3006
 
     cd ../virtual-product-app
-    build-and-run-container "virtual-product" 3009
+    build-and-run-container "cmr-virtual-product" 3009
 
     cd ../index-set-app
-    build-and-run-container "index-set" 3005
+    build-and-run-container "cmr-index-set" 3005
 
     cd ../indexer-app
-    build-and-run-container "indexer" 3004
+    build-and-run-container "cmr-indexer" 3004
 
     cd ../ingest-app
-    build-and-run-container "ingest" 3002
+    build-and-run-container "cmr-ingest" 3002
 
     cd ../dev-system
-fi
-
-if [[ $1 == "together" || -z $1 ]]; then
+elif [[ $1 == "together" || -z $1 ]]; then
   lein uberjar
-  docker build -t dev-system .
-  docker run -d -p 2999:2999 -p 3001:3001 -p 3002:3002 -p 3003:3003 -p 3004:3004 -p 3005:3005 -p 3006:3006 -p 3007:3007 -p 3008:3008 -p 3009:3009 -p 3010:3010 -p 3011:3011 -p 9210:9210 dev-system
+  docker build -t cmr-dev-system .
+  docker run -d -p 2999:2999 -p 3001:3001 -p 3002:3002 -p 3003:3003 -p 3004:3004 -p 3005:3005 -p 3006:3006 -p 3007:3007 -p 3008:3008 -p 3009:3009 -p 3010:3010 -p 3011:3011 -p 9210:9210 cmr-dev-system
   exit 0
+else
+  cd ../${1}-app
```

Comment:
This way of starting a single app needs to be documented.

Reply:
Good catch (didn't even know the others *were* documented ...)

Reply:
Done.